src/components/MenuButton.ts

```diff
@@ -0,0 +1,27 @@
+import { CSSProperties } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { Clickable, ClickableProps } from 'src/components/common'
+import colors from 'src/libs/colors'
+import { forwardRefWithName } from 'src/libs/react-utils'
+
+
+type MenuButtonProps = ClickableProps<keyof JSX.IntrinsicElements>
+
+const MenuButtonInternal = ({ disabled, children, ...props }: MenuButtonProps, ref) => {
```

Comment:
Thankfully this component is getting good unit test coverage from other unit tests that are consuming it... but we will want to keep an eye out for components that could use more unit testing as we make these improvements.